pyiron_continuum/fenics/job/generic.py

```diff
@@ -132,61 +127,46 @@ def __init__(self, project, job_name):
         self._python_only_job = True
         self._plot = Plot(self)
 
-        self.input = DataContainer(table_name='input')
-        self.input.mesh_resolution = 2
+        self.input.boundaries = BoundaryConditions()
+        self.input.mesh = Mesh(
+            'BoxMesh(p1, p2, nx, ny, nz)',
```

Comment:
Does the user have to know the syntax themselves? Not sure how to know how to set other meshes

Reply:
Yeah, the intent was to "allow" fenics-users to fill in what they're familiar with, and just store the serializable string as a super thin interface layer. The advantage is that then fenics gets all the storage and remote execution benefits. 

I did make one wrapper function for circular meshes. Not sure how I feel about it.

Definitely one improvement would be to make the parser's known elements easier to see.

Reply:
> I did make one wrapper function for circular meshes. Not sure how I feel about it.

As long as it is done this way in the software, I'd say we have no way around it, but I'd say that's perfectly fine. It's just on the pyiron side we should (at least at some point) make sure that the user can see what are the available options and what are the parameters they have to set.

Reply:
Available via `job.input.mesh.known_elements`, which is now introduced in both demo notebooks